ssg/yaml.py

```diff
@@ -46,6 +47,16 @@ def _get_implied_properties(existing_properties):
             "pkg_system" not in existing_properties):
         pkg_manager = existing_properties["pkg_manager"]
         result["pkg_system"] = PKG_MANAGER_TO_SYSTEM[pkg_manager]
+
+    if "uid_min" not in existing_properties:
+        result["uid_min"] = DEFAULT_UID_MIN
+
+    if "auid" not in existing_properties:
+        if "uid_min" in existing_properties:
+            result["auid"] = existing_properties["uid_min"]
```

Comment:
there is a much nicer way to do this in Python, existing_properties.get("uid_min", DEFAULT_UID_MIN). It's much simpler and shorter